bpf/http_sock.h

```diff
@@ -136,6 +136,22 @@ static __always_inline void process_http_response(http_info_t *info, unsigned ch
     info->status += (buf[RESPONSE_STATUS_POS + 2] - '0');
 }
 
+static __always_inline void finish_http(http_info_t *info) {
```

Comment:
This refactor wasn't really necessary after all. I had some temporary changes were I was closing http from the SSL code, but I didn't end up using it at the end.